flight/Modules/VtolPathFollower/vtol_follower_control.c

```diff
@@ -627,6 +529,142 @@ int32_t vtol_follower_control_attitude(float dT)
 	return 0;
 }
 
+
+static float loiter_deadband(float input) {
+	const float CMD_THRESHOLD = 0.2f;
+
+	if (input > CMD_THRESHOLD) {
+		input -= CMD_THRESHOLD;
+	} else if (input < -CMD_THRESHOLD) {
+		input += CMD_THRESHOLD;
+	} else {
+		input = 0;
+	}
+
+	input /= (1 - CMD_THRESHOLD);	// Normalize to -1 to 1 range.
+
+	/* Confine to desired range */
+	if (input > 1.0f) {
+		input = 1.0f;
+	} else if (input < -1.0f) {
+		input = -1.0f;
+	}
+
+	return expo3(input, 40);	// And apply 40% expo
+}
+
+/**
+ * Receives loiter commands and makes appropriate adjustments.
+ * @param[in] dT the time since last evaluation
+ * @param[in,out] hold_pos the hold position
+ * @param[out] att_adj an adjustment to be made to attitude for responsiveness.
+ */
+
+bool vtol_follower_control_loiter(float dT, float *hold_pos, float *att_adj) {
+	LoiterCommandData cmd;
+	LoiterCommandGet(&cmd);
+
+	// XXX TODO reproject when we're not issuing body-centric commands
```

Comment:
It's more of a marker for the correct place, and this is the next work item in the sequence of stuff to do.